scripts/inference/endpoint_generate.py

```diff
@@ -0,0 +1,221 @@
+# Copyright 2022 MosaicML LLM Foundry authors
```

Comment:
Please clearly identify this script as experimental and subject to change at any time at the top of the file